ee/agent/startup/startup.go

```diff
@@ -0,0 +1,94 @@
+// Package startup provides access to and manages storage of startup data:
+// flags/config values/settings that launcher needs during initialization,
+// before the knapsack is available.
+package startup
```

Comment:
I _suspect_ this is broader than `startup`. I'm thinking `shared`.

Though I think it might make more sense to interface with this through knapsack. storage/sqlite conforms to our store interface right? (Or do you think we need this before we have knapsack?)

Reply:
Thinking about it, an argument _against_ knapsack, is that something like `startup.GetValue()` might be cleaner than `knapsack.GetStartupValue` Maybe.

Reply:
> I suspect this is broader than startup. I'm thinking shared.

I know we talked on slack about this too -- I do like `shared` better but I think we both felt like it was a little broad. Maybe -- `shared data`? `shared settings`?

> storage/sqlite conforms to our store interface right?

It mostly does -- I didn't implement `Delete`, `DeleteAll`, or `ForEach` since I didn't need them yet.

> Or do you think we need this before we have knapsack?

Yup, exactly -- we'll need to be able to fetch values out of this in main.go, before the knapsack exists.

Reply:
Now that we've named the database table `startup_settings` I feel like switching to `shared` here is weird...do we want to do `shared_settings` or `startup_settings` in both places?